src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -73,12 +73,14 @@ public static bool IsCloseParenInParameterList(this SyntaxToken token)
 
         public static bool IsOpenParenInArgumentList(this SyntaxToken token)
         {
-            return token.Kind() == SyntaxKind.OpenParenToken && token.Parent.Kind() == SyntaxKind.ArgumentList;
+            return token.Kind() == SyntaxKind.OpenParenToken &&
+                (token.Parent.IsKind(SyntaxKind.ArgumentList) || token.Parent.IsKind(SyntaxKind.AttributeArgumentList));
         }
```

Comment:
Got it, I think I was confused between this and classification, where we have to be careful before checking parents of tokens, since sometimes we just classify a list of tokens without a tree.